web/routes/admin/personal-information.js

```diff
@@ -0,0 +1,31 @@
+const router = require('express').Router()
+const axios = require('axios')
+const url = require('../../settings.js')
+const middleware = require('../middleware/middleware.js')
+const role = 'ADMIN'
+
+router.get('/gegevens', middleware.authRole(role), function (req, res) {
+    const get_info = `${url}/users/userinfo/`
+    const token = req.signedCookies.Authorization
+
+    axios
+        .get(get_info, {
+            headers: {
+                Authorization: `${token}`,
+            },
+        })
+        .then((response) => {
+            const isNullField = Object.values(response.data).some(
```

Comment:
```suggestion
            const isNullField = Object.values(response.data).some(
```

Dit ben je denk ik vergeten te fixen, aangezien de iban veld niet wordt laten zien aan de admin hoor je dat niet mee te tellen bij de nullField check, maar dit gebeurt hier nog steeds

Reply:
als qucik fix stuur ik '-' naar de database omdat admins geen iban horen te hebben.

Reply:
Dan kunnen we hem zo nog laten